github-app/github-checks-app-gen.py

```diff
@@ -0,0 +1,52 @@
+import logging
+import sys
+from secrets import token_urlsafe
+from urllib.parse import urlencode, urlparse
+
+
+def init_logger():
+    logger = logging.getLogger(__name__)
+    logger.setLevel(logging.INFO)
+    h = logging.StreamHandler(sys.stdout)
+    logger.addHandler(h)
+
+    return logger
+
+
+logger = init_logger()
+
+
+if __name__ == '__main__':
+    print("Please select an option cloud or enterprise server:")
+    print("1. GitHub Cloud")
+    print("2. GitHub Enterprise Server")
+    choice = input("Enter 1 or 2: ")
+
+    name = 'Hashicorp Vault Radar Test App'
```

Comment:
nit: `HashiCorp`

Reply:
Oh also this will be used in their production, should we remove the `Test` ? I think ours on Cloud is called `HashiCorp Vault Radar Checks App` ?